tests/Fusonic/Linq/Test/MiscTest.php

```diff
@@ -36,19 +32,25 @@ public function testConcat_ReturnsConcatenatedElements()
         $this->assertEquals("d", $all[3]);
     }
 
-    public function testConcat_ThrowsArgumentExceptionIfNoTraversableArgument()
+    /**
+     * @expectedException InvalidArgumentException
```

Comment:
@voorproever  I am ok with these changes although personally I`d avoid the attribute approach. 

So now need for changes ... just a general remark ;)